impeller/geometry/geometry_unittests.cc

```diff
@@ -2090,5 +2092,38 @@ TEST(GeometryTest, Gradient) {
   }
 }
 
+TEST(GeometryTest, HalfConversions) {
+#ifdef FML_OS_WIN
+  GTEST_SKIP() << "The reason this doesn't work on Windows";
```

Comment:
Perhaps "Half-precision floats (IEEE 754) are not portable and unavailable on Windows."

Reply:
We could write or use our own library (or a header rather) for doing this portably.

Reply:
We could, one exists in an earlier commit. however we have as of today no use for it as this is only used on metal backends.

Reply:
Comment fixed